find_parameters_by_coordinates.py

```diff
@@ -0,0 +1,50 @@
+import constants
+import math
+
+
+def measure_distance(
+    y, x, picture_shape, camera_height, y_leaning_angle, x_turning_pixels
+):
+    height, width, _ = picture_shape
+
+    y_center = height / 2
+    # the center x for the calculations is not really the center pixel
+    x_center = int(width / 2) - x_turning_pixels
+
+    dy = y - y_center

```

Comment:
a negative angle is fine for us because we have the camera turning angle that keeps it above 0 and we need that negative because if we use it with abs it will always be considered more down then it should have